.github/workflows/bbw_build_container.yml

```diff
@@ -72,9 +72,6 @@ jobs:
             image: ubuntu:23.10
             branch: 10.11
             platforms: linux/amd64, linux/arm64/v8
-          - dockerfile: fedora.Dockerfile
-            image: fedora:37
-            platforms: linux/amd64, linux/arm64/v8
```

Comment:
why does fedora:37 need to be removed? It's EOL and this wasn't previously? If yes, I think it should be in a separate commit

Reply:
https://github.com/MariaDB/buildbot/pull/283 ok, will rebase on main